server/cache.go

```diff
@@ -384,6 +386,9 @@ func (c *clusterInfo) handleRegionHeartbeat(region *core.RegionInfo) error {
 		if len(origin.DownPeers) > 0 || len(origin.PendingPeers) > 0 {
 			saveCache = true
 		}
+		if region.ApproximateSize != origin.ApproximateSize {
+			saveCache = true
```

Comment:
do we need save here? 

@disksing

Reply:
Yes. Schedulers need it update in time.

Reply:
so I guess every heartbeat will save to etcd, this may reduce the performance.

Reply:
They are only saved to memory cache, and they won't update every time. We store region size in MB, please see https://github.com/pingcap/pd/pull/814/files#diff-1f3079039053bbe65b3d8b75ab87d5ceR325